pkg/utils/key.go

```diff
@@ -22,11 +24,11 @@ func ParseKey(format, key string) ([]byte, error) {
 	case "hex":
 		key, err := hex.DecodeString(key)
 		if err != nil {
-			return nil, errors.WithStack(err)
+			return nil, errors.Trace(err)
 		}
 		return key, nil
 	}
-	return nil, errors.New("unknown format")
+	return nil, errors.Annotate(berrors.ErrUnknown, "unknown format")
```

Comment:
`ParseKey` are only used to decode the CLI args, they should be annotated with `ErrInvalidArgument`.